openvino_tensorflow/api.cc

```diff
@@ -28,19 +28,18 @@ void enable() { Enable(); }
 void disable() { Disable(); }
 bool is_enabled() { return IsEnabled(); }
 
-bool check_backend(char* backend) {
+bool check_backend(std::string backend) {
```

Comment:
pls revert it back to ctype function argument style

Reply:
Done